multiversx_sdk/abi/serializer.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, List, Sequence
+
+from multiversx_sdk.abi.codec import Codec
+from multiversx_sdk.abi.parts import PartsHolder
+from multiversx_sdk.abi.values_multi import *
+
+
+class Serializer:
+    """
+    The serializer follows the rules of the MultiversX Serialization format:
+    https://docs.multiversx.com/developers/data/serialization-overview
+    """
+
+    def __init__(self, parts_separator: str):
```

Comment:
maybe `parts_separator` can be `@` by default? But i also understand this respects the Go implementation. Not a big deal anyway.

Reply:
The `@` separator is provided in the higher-level components. For example: 
 - https://github.com/multiversx/mx-sdk-py/blob/codecs-init/multiversx_sdk/core/transactions_factories/smart_contract_transactions_factory.py#L45